datasetviewer/mainview/MainViewPresenter.py

```diff
@@ -31,6 +33,7 @@ def __init__(self, mainview, *subpresenters):
         self._preview_presenter = None
```

Comment:
I think either omit these instance fields from the declaration OR add a comment above them saying that the next line `presenters.register_master` should complete the initialisation.

Also, how do we prevent the MainViewPresenter being badly initialised. i.e. I forget to add one sub-presenter to the list. Presumably this would cause a hard-to-find problem further down the line?